fix: fixing chat on the dashboard

sostene12 opened the pull request:
### What does this PR do?
- implement fix on dashboard
### Description 
- implement  fix chat on  vendor and admin dashboard
- implement fix on redirect of the user
#### How should this be manually tested?
- clone the repo using git clone repo's URL
- change branch `git  checkout fix-dashboard-chat-105`
- create .env file  and paste variables from .env.example
- run `npm run dev`
- try login as a any user and test chat

wakaflorien committed:
Merge pull request #46 from atlp-rwanda/fix-logout-cp-103

fix: fixing logout  and login

sostene12 committed:
fix(dashboard-chart): vendor and admin

- implement vendor and admin chat

[Finished fix-dashboard-chat-105]

sostene12 committed:
fix(dashboard-chat): admin and vendor chat

- implement vendor and dashboard chat

[Finished fix-dashboard-chat-105]

Shumba250 reviewed (approved):
checkout seems not to be working as well as logging in with google

sostene12 committed:
fix(chat): dashboard chat

- implement dashboard chat and rebased to develop

[Finished fix-dashboard-chat-105]
